radio/src/lua/api_general.cpp

```diff
@@ -1535,7 +1535,20 @@ static int luaGetUsage(lua_State * L)
   lua_pushinteger(L, instructionsPercent);
   return 1;
 }
+/*luadoc
```

Comment:
CR before

Reply:
do I take away the luadoc portion until it is approved?

Reply:
No no, it's fine. Since it is purely optional and informative (ie not affecting existing scripts), you could do a 2.3.x version if this is usefull to you

Reply:
Should I rebase this to 2.3 branch?